lib/s3_meta_sync/syncer.rb

```diff
@@ -210,12 +222,12 @@ def download_meta(destination)
       parse_yaml_content(content)
     rescue OpenURI::HTTPError
       retries ||= 1
-      if retries == 2
-        raise RemoteWithoutMeta
-      else
+      if retries <= 1
```

Comment:
that is inconsistent with the other retry block though, I'll move the increment outside the if/else block